src/main/resources/templates/hint/list.html

```diff
@@ -25,6 +26,9 @@ <h2>Šifra <span th:text="${cypher.name}">cypherName</span></h2>
         <td th:text="${cypher.stage}" align="center">stage</td>
         <td th:text="${cypher.location}">X, Y</td>
         <td th:text="${cypher.codeword}">codeword</td>
+        <td align="center">
+            <a href="#" th:href="@{'/progress?cypherId='+${cypher.cypherId}}">Stav</a>
```

Comment:
Tohle by myslím mělo jít zapsat nějak takhle:
@{/progress/ (cypherId=${cypher.cypherId})}

Je to většinou přehlednější a mělo by to skládat validní URL.

Link: https://www.thymeleaf.org/doc/articles/standardurlsyntax.html
Kapitola 5, Adding parameters

Reply:
upraveno a otestovano